src/main/java/org/myrobotlab/service/Arduino.java

```diff
@@ -471,6 +472,26 @@ public void reattach(DeviceMapping dm) {
       if (servo.isEnabled()) {
         msg.servoAttachPin(dm.getId(), pin);
       }
+    } else if (attachable instanceof UltrasonicSensorControl) {
+      // reattach logic

```

Comment:
I think there needs to be a merge with develop ... there's a missing log message here.